db/sql/sql.go

```diff
@@ -876,8 +877,8 @@ func buildSelect(sc *selectContext) (string, []interface{}, error) {
 			}
 		}
 
-		if sc.paginator.Limit > 0 {
-			q = q.Limit(sc.paginator.Limit)
+		if sc.paginator.Limit != math.MaxUint64 {
```

Comment:
Same change should be necessary for `file.go` (sqlite3)

Reply:
Right, fixed